### ISSUE TITLE
Ability to set the default image size and proportion 

### ISSUE THREAD

**Issue by harbolaez** [aspect-apps/react-native-images-collage#28]
Want to be able to load an image base on the way it was left off.

I'm using `collage.current.refs` and saving the state of each of the imageIds. It works fine for the first imageId but not for the rest. 

Looking to see what is a better way to do this. @lukebrandonfarrell  any suggestion?

**Comment by lukebrandonfarrell** [comment]
@harbolaez Do you mean editing a collage, saving it, then opening it up for editing again?

**Comment by harbolaez** [comment]
@lukebrandonfarrell correct. I'm working on something, I can send it over later. It seems to be good for my needs.
